.github/ISSUE_TEMPLATE/test-plan.md

```diff
@@ -2,36 +2,38 @@
 name: Test Plan
 about: Create a new manual test plan meta issue
 labels: "test-plan"
-
 ---
 
 # Manual Test Plan
 
-When picking up a test case, please add your name to this overview beforehand and tick the checkbox when finished.
-Testing can be started when the first build candidate (BC) is available in the CFT region.
+- When picking up a test case, please add your name to this overview beforehand and tick the checkbox when finished.
+- Testing can be started when the first build candidate (BC) is available in the CFT region.
+- For each repository, update the compare version range to get the list of commits to review.
 
-## Smoke Testing ESS setup
+## ES apm-data plugin
 
-Thanks to https://github.com/elastic/apm-server/issues/8303 further smoke tests are run automatically on ESS now.
-**Consider extending the smoke tests to include more test cases which we'd like to cover**
+<!-- Add any issues / PRs which were worked on during the milestone release https://github.com/elastic/elasticsearch/tree/main/x-pack/plugin/apm-data-->
 
-## go-docappender library
+## apm-aggregation
 
-<!-- Add any issues / PRs which were worked on during the milestone release https://github.com/elastic/go-docappender/pulls-->
+<!-- Add any issues / PRs which were worked on during the milestone release https://github.com/elastic/apm-aggregation/pulls-->
 
-## apm-data library
+List of changes: https://github.com/elastic/apm-aggregation/compare/v1.2.0...v1.3.0
 
-<!-- Add any issues / PRs which were worked on during the milestone release https://github.com/elastic/apm-data/pulls-->
+## go-docappender
 
+<!-- Add any issues / PRs which were worked on during the milestone release https://github.com/elastic/go-docappender/pulls-->
 
-## Test cases from the GitHub board
+List of changes: https://github.com/elastic/go-docappender/compare/v2.4.0...v2.10.0
+
+## apm-data
+
+<!-- Add any issues / PRs which were worked on during the milestone release https://github.com/elastic/apm-data/pulls-->
 
-<!-- Replace MAJOR.MINOR with the appropriate versions below -->
-<!-- Label the relevant MAJOR.MINOR Issues / PRs with the `test-plan` label https://github.com/elastic/apm-server/issues?page=1&q=-label%3Atest-plan+label%3AvMAJOR.MINOR.0+-label%3Atest-plan-ok-->
-<!-- [apm-server MAJOR.MINOR test-plan](https://github.com/elastic/apm-server/issues?q=is%3Aissue+label%3Atest-plan+-label%3Atest-plan-ok+is%3Aclosed+label%3AvMAJOR.MINOR.0) -->
```

Comment:
tbh I actually quite like this way of automatically pulling apm-server PRs with the right labels. Can we keep this?